Modules/HRV_DFA/main.cpp

```diff
@@ -0,0 +1,200 @@
+#include "DFA.h"
+
+using namespace std;
+
+
+
+int main()
```

Comment:
To nie powinno się tutaj znaleźć.

Reply:
[ISSUE] - Jeżeli działanie algorytmu wymaga przeniesienia części jego logiki poza klasę zawierającą narusza to istotnie hermetyzację. Klasa powinna zawierać całą implementację wewnątrz i nie uzewnętrzniać szczegółów implementacji.

Reply:
Algorytm będzie wymagał drobnej implementacji poza klasą. Jak będę mieć uporządkowany kod, to w mainie zawrę informację, które elementy muszą się znaleźć w aplikacji głównej (głównie chodzi o input i wybór przedziałów)